src/Gui/Stylesheets/Dark-blue.qss

```diff
@@ -497,7 +498,7 @@ QTabBar {
     background-color: transparent;
 }
 
-/* Workaround for QTabBars created from docked QDockWidgets which don't draw the border if not set and reset as follows: */
+/* Workaround for QTabBars created from docked QDockWidgets which don't draw the border if not set and reseted as follows: */

```

Comment:
Just a heads up, in English, the past and participle of "set" is "set". Same with "reset".

Reply:
Thanks for the fix =)

Reply:
I didn't change anything. What I mean is, you have to change "reseted" to "reset" :)

Reply:
OK, I thought you already included the fix.
I'll do it then, of course.
Thanks again